app/assets/stylesheets/_direct_assessment.scss

```diff
@@ -0,0 +1,3 @@
+.selectize-input.not-full > input {
+  width: 100% !important;
```

Comment:
This is very sad but https://github.com/brianreavis/selectize.js/issues/238

Otherwise the placeholder gets cut off.